chartfx-chart/src/main/resources/de/gsi/chart/chart-icons.css

```diff
@@ -0,0 +1,59 @@
+@font-face {
+  font-family: 'fair-chart-icons';
+  src:  url('fonts/fair-chart-icons.eot?yr8ymj');
+  src:  url('fonts/fair-chart-icons.eot?yr8ymj#iefix') format('embedded-opentype'),
+    url('fonts/fair-chart-icons.ttf?yr8ymj') format('truetype'),
+    url('fonts/fair-chart-icons.woff?yr8ymj') format('woff'),
+    url('fonts/fair-chart-icons.svg?yr8ymj#fair-chart-icons') format('svg');
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
+
+[class^="icon-"], [class*=" icon-"] {
+  /* use !important to prevent issues with browser extensions that change fonts */
+  font-family: 'fair-chart-icons' !important;
```

Comment:
Codacy found an issue: [Use of !important](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5744990)